src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlFileGenerator.cs

```diff
@@ -2817,6 +2817,11 @@ private string GetCastString(INamedTypeSymbol targetProperty, XamlObjectDefiniti
             }
             return "";
         }
+        
+        private INamedTypeSymbol GetUnderlyingType(INamedTypeSymbol propertyType)
```

Comment:
`FindUnderlyingType` or `TryFindUnderlyingType` would be more appropriate here, as it's not a general case.

Reply:
OK, I chose `FindUnderlyingType` (I prefer than `TryFindUnderlyingType` because TryFind makes think that it returns a bool).